### PR TITLE
feat: Preserve leading spaces in mp_ds2cards.sas.   Closes #16

### LINKED ISSUE #16: Preserve leading spaces in mp_ds2cards
`mp_ds2cards` strips all spaces out of character values while leading spaces should be preserved, since they are important.

Let's consider the following dataset:

```sas
data test;
    a = '   123';
    b = 123;
    c = '1"23';
    d = 'ww''"ww';
    e = 123.4;
    f = '123.4';
run;
```

The leading spaces in `a` must be preserved. Currently macro generates the following datalines:

```sas
data work.test ;
attrib 
a                                length= $6
b                                length= 8
c                                length= $4
d                                length= $6
e                                length= 8
f                                length= $5
;
infile cards dsd delimiter=',';
input 
   a                                :$char.
   b
   c                                :$char.
   d                                :$char.
   e
   f                                :$char.
;
datalines4;
123,123,'1"23',"ww'""ww",123.4,123.4
;;;;
run;
```

What I would expect is the following:

```sas
data work.test ;
attrib 
a                                length= $6
b                                length= 8
c                                length= $4
d                                length= $6
e                                length= 8
f                                length= $5
;
infile cards dsd delimiter=',';
input 
   a                                :$char.
   b
   c                                :$char.
   d                                :$char.
   e
   f                                :$char.
;
datalines4;
"   123",123,'1"23',"ww'""ww",123.4,123.4
;;;;
run;
```

Let me know if I should submit a PR implementing these changes

### PR DESCRIPTION
**Pull request by sspkmnd**
Fixes #16.

### CODE REVIEW

File: mc_all.sas [hunk 1]
```diff
@@ -1989,10 +1989,10 @@ data datalines_2;
     ('PROCESSED_DTTM','VALID_FROM_DTTM','VALID_TO_DTTM')));
   if type='num' then dataline=
     cats('ifc(int(',name,')=',name,'
-      ,put(',name,',best32.)
-      ,substrn(put(',name,',bestd32.),1
-      ,findc(put(',name,',bestd32.),"0","TBK")))');
-  else dataline=cats('strip(',name,')') ;
+      ,put(',name,',best32.-l)
+      ,substrn(put(',name,',bestd32.-l),1
+      ,findc(put(',name,',bestd32.-l),"0","TBK")))');
```
Comment (allanbowe) [inline mc_all.sas:1994]: Nice, hadn't seen the `-l` option before!  Does it work on 9.3?

Comment (allanbowe) [reply mc_all.sas:1994]: yes - back to 9.2.  Will left align:  https://support.sas.com/documentation/cdl/en/lrdict/64316/HTML/default/viewer.htm#a000199354.htm

File: mc_all.sas [hunk 2]
```diff
@@ -2082,7 +2082,7 @@ data _null_;
 /* remove all formats for write purposes - some have long underlying decimals */
   format _numeric_ best30.29;
   length __dataline $32767;
-  __dataline=catq('cqsm',&datalines);
+  __dataline=catq('cqtmb',&datalines);
```
Comment (allanbowe) [inline mc_all.sas:2085]: so we are left aligning the numerics, and otherwise keeping the leading *b*lanks.  Looks great!

**Review by allanbowe — APPROVED** [review]

File: mc_all.sas (hunk 1 above)
Comment (sspkmnd) [reply mc_all.sas:1994]: yes, left align values, so later we use `t` for trim in CATQ function instead of `s`

**Pull request merged**